dipy/tracking/utils.py

```diff
@@ -956,3 +958,124 @@ def reduce_rois(rois, include):
             exclude_roi |= rois[i]
 
     return include_roi, exclude_roi
+
+def flexi_tvis_affine(sl_vox_order, grid_affine, dim, voxel_size):
+    """ Computes the mapping from voxel indices to streamline points,
+        reconciling streamlines and grids with different voxel orders
+
+    Parameters
+    ----------
+    sl_vox_order: string of length 3
+        a string that describes the voxel order of the streamlines (ex: LPS)
+    grid_affine: nii_aff: array (4, 4),
```

Comment:
I'm interpreting this as "why do you include both the sl_vox_order and the grid_affine as parameters?" The streamline voxel order doesn't necessarily match that of the affine of the underlying grid one is trying to target/display in fvtk... the function called here (get_flexi_tvis_affine) converts both to orientations, then reconciles them with an affine.  The orientations can be different, so must be pulled from both the nifti and the trk headers.